aws/resource_aws_mq_broker_test.go

```diff
@@ -325,7 +325,109 @@ func TestAccAwsMqBroker_allFields(t *testing.T) {
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "configuration.#", "1"),
+					resource.TestMatchResourceAttr("aws_mq_broker.test", "configuration.0.id", regexp.MustCompile(`^c-[a-z0-9-]+$`)),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "configuration.0.revision", "2"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "deployment_mode", "ACTIVE_STANDBY_MULTI_AZ"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "engine_type", "ActiveMQ"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "engine_version", "5.15.0"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "host_instance_type", "mq.t2.micro"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "maintenance_window_start_time.#", "1"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "maintenance_window_start_time.0.day_of_week", "TUESDAY"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "maintenance_window_start_time.0.time_of_day", "02:00"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "maintenance_window_start_time.0.time_zone", "CET"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "publicly_accessible", "true"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "security_groups.#", "2"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "subnet_ids.#", "2"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "user.#", "2"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "user.1344916805.console_access", "true"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "user.1344916805.groups.#", "3"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "user.1344916805.groups.2456940119", "first"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "user.1344916805.groups.3055489385", "second"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "user.1344916805.groups.607264868", "third"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "user.1344916805.password", "SecondTestTest1234"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "user.1344916805.username", "SecondTest"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "user.3793764891.console_access", "false"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "user.3793764891.groups.#", "0"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "user.3793764891.password", "TestTest1234"),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "user.3793764891.username", "Test"),
+					resource.TestMatchResourceAttr("aws_mq_broker.test", "arn",
+						regexp.MustCompile("^arn:aws:mq:[a-z0-9-]+:[0-9]{12}:broker:[a-z0-9-]+:[a-f0-9-]+$")),
+					resource.TestCheckResourceAttr("aws_mq_broker.test", "instances.#", "2"),
+					resource.TestMatchResourceAttr("aws_mq_broker.test", "instances.0.console_url",
+						regexp.MustCompile(`^https://[a-f0-9-]+\.mq.[a-z0-9-]+.amazonaws.com:8162$`)),
```

Comment:
this does assume we're only running on AWS Public and not Government/China etc?

Reply:
For now, yes - probably more than half of all our acceptance tests would fail in gov/cn region anyways because many services just aren't supported at all. It's something on my list to address more systematically, but not very high atm.